install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eux
+
```

Comment:
need to check OS version first

Reply:
also help message and error messages are needed

a simple example: https://github.com/CshlSiepelLab/phast/blob/master/binary_install.sh

we can follow similar style to support both debian (ubuntu) and centos